### ISSUE TITLE
Smoother loading and transitions

### ISSUE THREAD

**Issue by timothyachumba** [theloversmag/.com#56]
Hey @Slgoetz, do you think we could add a smoother loading? In the past i've used the css animation opacity to 1 when page loads but I wonder if there something better we could do? 

I guess just having the page fade in when everything is ready would be nice, rather than things jumping as the JS load.

Do you think this is something you could add before Tuesday's launch?

**Comment by Slgoetz** [comment]
@timothyachumba  let me know if you like this loading effect on the title and I can add it in a few places.  

Side note, we need to clean up the JS a bit.

**Comment by timothyachumba** [comment]
@Slgoetz Yeah, this is cool! It would work pretty well on the single interview page too I think.

Something that would be good to hide would be the page rendering, maybe with a simple page load.

![Kapture 2020-11-30 at 15 05 49](https://user-images.githubusercontent.com/2601332/100626338-9d3cae80-331d-11eb-80c7-2e5ddbf33f00.gif)

Happy to clean up the JS. What do we need to do?
